stories/utils/ScreenshotReadyWrapper.tsx

```diff
@@ -0,0 +1,78 @@
+// (C) 2007-2019 GoodData Corporation
+import * as React from 'react';
+
+export interface IScreenshotReadyWrapperProps {
+    resolver: (element: Element) => boolean;
+    interval?: number;
+}
+
+export interface IScreenshotReadyWrapperState {
+    ready: boolean;
+}
+
+export class ScreenshotReadyWrapper extends
+    React.Component<IScreenshotReadyWrapperProps, IScreenshotReadyWrapperState> {
+
+    public static defaultProps = {
+        interval: 200
+    };
+
+    private timer: any = null;
+    private componentRef: any = null;
+
+    constructor(props: IScreenshotReadyWrapperProps) {
+        super(props);
+
+        this.state = { ready: false };
+        this.componentRef = (React as any).createRef();
```

Comment:
I guess this is an issue with `@types/react` which has a version `16.0.34` but in react-components we are already using `"react": "16.5.2"` I would say this should be in sync and it solves this `any` issues

Reply:
as explained this have to be in sync with typescript version....